docs/config/launch.markdown

```diff
@@ -49,6 +49,44 @@ to edit your wezterm configuration:
 wezterm start -- vim ~/.wezterm.lua
 ```
 
+## Specifying the current working directory
+
+If you'd like `wezterm` to start running a program in a specific working
+directory you can do so via the config, CLI, and when using
+[`SpawnCommand`](lua/SpawnCommand.md):
+
+* Setting the [`default_cwd`](lua/config/default_cwd.md) via the config:
+
+  ```lua
+  return {
+    default_cwd = "/some/path",
+  }
+  ```
+
+* One off program in a specific working directory via the CLI:
+
+  ```bash
+  wezterm start --cwd /some/path
+  ```
+
+* Using `SpawnCommand` in a specific working directory:
```

Comment:
I'd tweak this to link to where/how the user might do this:

The [SpawnCommandInNewTab](https://wezfurlong.org/wezterm/config/lua/keyassignment/SpawnCommandInNewTab.html), [SpawnCommandInNewWindow](https://wezfurlong.org/wezterm/config/lua/keyassignment/SpawnCommandInNewWindow.html), [SpawnTab](https://wezfurlong.org/wezterm/config/lua/keyassignment/SpawnTab.html) key assignments, and the [Launcher Menu](https://wezfurlong.org/wezterm/config/launch.html#the-launcher-menu) described below all accept a [SpawnCommand](https://wezfurlong.org/wezterm/config/lua/SpawnCommand.html) object that accepts an optional `cwd` field